### ISSUE TITLE
Add support to prioritize repositories

### ISSUE THREAD

**Issue by CloneWith** [Droid-ify/client#713]
**Problem**
Due to my unstable Internet connection, some repositories are available, but not all of them are as fast at the same time.
Then prioritizing them would be a good idea to somehow avoid such problems, since:

- When multiple repositories have the same version of an app, the one with a higher priority is shown and to download from first.
- Otherwise, new versions go first.

**Solution**
Add a manual sorting function to the repository list. Users can simply drag them up and down to prioritize them.

**Alternative**
Add an option to select the *major repository*, and always fetch APKs from this repo first (not so good but simpler).

**Comment by Uneccessary** [comment]
Would be also useful if you prefer to receive updates from the developer's repo, or IzzyOnDroid e.g. quicker updates instead of F-Droid which is currently beeing always prioratized.

**Comment by palashman** [comment]
I agree with ideas  (1) to have ability to rearrange repos and (2) to prioritize owner particular repos.
.
For example, I have issues with Izzi (stuck download process), but in the suggestion list it stands at the 1st place.
